cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,24 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('User', () => {
-  before(() => {
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
 
+describe('User', () => {
+  let user;
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
   });
```

Comment:
You should use the generated user to register. Use the register function with parameters.